pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -325,61 +330,61 @@ func (s *Service) CanStartASGInstanceRefresh(scope *scope.MachinePoolScope) (boo
 
 // StartASGInstanceRefresh will start an ASG instance with refresh.
 func (s *Service) StartASGInstanceRefresh(scope *scope.MachinePoolScope) error {
-	strategy := ptr.To[string](autoscaling.RefreshStrategyRolling)
-	var minHealthyPercentage, maxHealthyPercentage, instanceWarmup *int64
+	strategy := ptr.To(autoscalingtypes.RefreshStrategyRolling)
+	var minHealthyPercentage, maxHealthyPercentage, instanceWarmup *int32
```

Comment:
I suspect it's not going to have an impact, but what's the reason for taking this from 64 to 32 bit?

Reply:
These integers changed in the AWS SDK to int32, I also tried to avoid introducing any API changes because it wouldn't be easy to convert